### DISCUSSION TITLE
Why isn't all of the package data included in the VS Code extension official install?

### DISCUSSION THREAD

**Discussion by yarakos95** [James-Yu/LaTeX-Workshop#4350] [Q&A] [answered]
The LaTeX Workshop repository contains package data files in two directories:

- `dev/packages/`
- `data/packaegs/`

The files in `data/packages/` are included in [the VS Code extension official install][vsmarketplace], but the files in `dev/packages/` are not included in the official install.

[vsmarketplace]: https://marketplace.visualstudio.com/items?itemName=James-Yu.latex-workshop "LaTeX Workshop - Visual Studio Marketplace"

Why are the files in <code>**dev**/packages/</code> not included in the official install?

Of course I know that I can add package data files in `latex-workshop.intellisense.package.dirs`, but as a user, it would be nice to have more package suggestions in the official install.

**Comment by James-Yu** [comment, marked as answer]
Too big in size. Only reason.

**Reply by yarakos95** [reply]
Thank you for your quick reply.

I was relieved for a simple reason. I was worried about what I would do if it was a complicated technical issue.
